frontend/src/pages/privacy.vue

```diff
@@ -58,13 +76,27 @@ export default defineComponent({
   layout: "content-layout",
   setup() {
     const i18n = useI18n()
+    const { $cookies } = useContext()
+    const featureFlagStore = useFeatureFlagStore()
 
     useMeta({
       title: `${i18n.t("privacy.title", {
         openverse: "Openverse",
       })} | Openverse`,
       meta: [{ hid: "robots", name: "robots", content: "all" }],
     })
+
+    const handleChange = ({ checked }: { checked: boolean }) => {
+      featureFlagStore.toggleFeature("analytics", checked ? ON : OFF)
+      $cookies.set("features", featureFlagStore.flagStateMap)
```

Comment:
Do you think moving the cookies handling to the store (inside the `toggleFeature` method) would be better? The UI store's settings use store to set the cookies.

Reply:
I did not know we could do `this.$nuxt.$cookies` in a store. I'll update this.